smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/impl/KafkaSource.java

```diff
@@ -150,6 +150,7 @@ public KafkaSource(Vertx vertx,
                     + 11_000L; // it's possible that it might expire 10 seconds before when we need it to
 
             kafkaConsumer.partitionsAssignedHandler(set -> {
+                commitHandler.partitionsAssigned(vertx.getOrCreateContext(), set);
```

Comment:
shouldn't we pause first?

Reply:
Works either way since we are in the context's event thread. I'll move it if it makes it clearer.